samples/nrf9160/http_application_update/README.rst

```diff
@@ -47,33 +47,28 @@ Specifying the image file
 =========================
 
 Before building the sample, you must specify where the image file will be located.
-If you do not want to host it yourself, you can upload it to nRF Cloud for testing purposes.
-See `Hosting your image on nRF Cloud`_ for instructions.
+If you do not want to host it yourself, you can upload it to a public S3 bucket on AWS.
+See `AWS S3 bucket setup`_ for instructions.
 
 To specify the location in |SES|:
 
 1. Select **Project** > **Configure nRF Connect SDK Project**.
 #. Navigate to **HTTP application update sample** and specify the **Download host name** (``CONFIG_DOWNLOAD_HOST``) and **The file to download** (``CONFIG_DOWNLOAD_FILE``).
 #. Click **Configure** to save the configuration.
 
+.. include:: /includes/aws_s3_bucket.txt
 
-Hosting your image on nRF Cloud
--------------------------------
+Hosting your image on an AWS S3 Server
+--------------------------------------
 
-1. Go to `firmware.nrfcloud.com`_ and sign in.
-   If you already have an nRF Cloud account, you can use the same credentials.
+1. Go to `AWS S3 console`_ and sign in.
+   If you already have an AWS account, you can use the same credentials.
```

Comment:
delete this line

Reply:
Both from L65-L66?

Reply:
Haven't added this change yet since I was unsure what to remove.

Reply:
Only "If you already have an AWS account, you can use the same credentials."
Because of course you can. ;) This was pointed out in the context of nrfcloud vs. firmware.nrfcloud.com.

Reply:
okei :)